src/TorchSharp/NN/Upsample.cs

```diff
@@ -220,6 +226,21 @@ public override Tensor forward(Tensor tensor)
                 return new Tensor(res);
             }
 
+            public UpsampleMode mode { get; private set; }
+
+            public bool? align_corners { get; private set; }
+
+            public ReadOnlySpan<long> size {
+                get { return _size is null ? null : new ReadOnlySpan<long>(_size!); }
+            }
+
+            public ReadOnlySpan<double> scale_factor {
+                get { return _scale_factor is null ? null : new ReadOnlySpan<double>(_scale_factor!); }
+            }
+
+            private long[]? _size;
+            private double[]? _scale_factor;
```

Comment:
Is there a need for a separate field for the size and scale factor? Why not just store them as regular properties like the other fields?

Reply:
The C# compiler didn't like the ReadOnlySpan<T> as the property type when the getter/setters were automatically generated.